scheduler/ecs/ecs.go

```diff
@@ -743,16 +757,6 @@ func (e *LoadBalancerExposureError) Error() string {
 	return fmt.Sprintf("Process %s is %s, but load balancer is %s. An update would require me to delete the load balancer.", e.proc.Type, e.proc.Exposure, lbExposure)
 }
 
-// LoadBalancerPortMismatchError is returned when the port stored in the data store does not match the ELB instance port
-type LoadBalancerPortMismatchError struct {
```

Comment:
This error case is no longer possible, since the ELB determines the host port once it's been created.